docs/users.md

```diff
@@ -314,6 +315,83 @@ To create a control plane of type `host` run the command:
 kflex create cp4 --type host
 ```
 
+To create a control plane of type `external` with the required options, run the command: 
+
+```shell
+kflex adopt --adopted-context <kubeconfig-context-of-external-cluster> cp5
+```
+
+*Important*: This command generates a secret containing a long-lived token for accessing 
+the external cluster within the namespace associated with the control plane. The secret is automatically 
+removed when the associated control plane is deleted.
+
+### Creating a control plane of type `external` with the API
+
+To create a control plane of type `external` with the API, you need to provide 
+first a **bootstrap secret** containing a bootstrap Kubeconfig for accessing the external cluster.
+The bootstrap Kubeconfig is used by the KubeFlex controllers to generate a long-lived
+token for accessing the external cluster.  The bootstrap kubeconfig is required to have only one context, 
+so given a Kubeconfig for the external cluster `$EXTERNAL_KUBECONFIG` with context for the external
+cluster `$EXTERNAL_CONTEXT` you can generate the `$BOOTSTRAP_KUBECONFIG` with the command:
+
+```shell
+kubectl --kubeconfig=$EXTERNAL_KUBECONFIG config view --minify --flatten \
+--context=$EXTERNAL_CONTEXT > $BOOTSTRAP_KUBECONFIG
+```
+
+If the Kubeconfig for your external cluster uses a loopback address for the server URL, you
+need to follow these [steps](#determining-the-endpoint-for-an-external-cluster-using-loopback-address) 
+to determine the address to use for `cluster.server` in the Kubeconfig and set that value in
+the `$BOOTSTRAP_KUBECONFIG` created in the previous step. If the address is the value of `$INTERNAL_ADDRESS`
```

Comment:
The wording here is not quite on point; the address is not going into environment variable, it is going into the file.

Reply:
Updated to clarify that.